File: print-manager-dotnet-core-cs/PrintOptions.cs
using System;
using ceTe.DynamicPDF.Printing;

namespace print_manager_dotnet_core_cs
{
    class PrintOptions
    {
        public static void Run()
        {
            // Create a print job with the document to be printed to the default printer
            PrintJob printJob = new PrintJob(Printer.Default, Util.GetPath("Resources/DocumentA.pdf"), 2, 2);

            // Set resolution to high
            printJob.PrintOptions.Resolution = ResolutionList.High;

            // Set the print priority
            printJob.PrintPriority = 2;

            // Sent the print scaling
            printJob.PrintOptions.Scaling = new PercentagePageScaling(0.5f);

            // Set the page orientation
            printJob.PrintOptions.Orientation.Type = OrientationType.Portrait;

            // Set the option to print in color
            if (printJob.Printer.Color)
            {
                printJob.PrintOptions.Color = true;
            }

            // Print the print job
            printJob.Print();
        }
    }
}

File: print-manager-dotnet-core-vb/PrintOptions.vb
Imports ceTe.DynamicPDF.Printing

Public Class PrintOptions
    Public Shared Sub Run()
        ' Create a print job with the document to be printed to the default printer
        Dim MyPrintJob As PrintJob = New PrintJob(Printer.Default, Util.GetPath("Resources/DocumentA.pdf"), 3, 3)

        ' PrintOptions to set Resolution
        MyPrintJob.PrintOptions.Resolution = ResolutionList.High

        ' PrintOptions to set PrintPriority
        MyPrintJob.PrintPriority = 2

        ' PrintOptions to set Scaling
        MyPrintJob.PrintOptions.Scaling = New PercentagePageScaling(0.5F)

        ' PrintOptions to set Orientation
        MyPrintJob.PrintOptions.Orientation.Type = OrientationType.Portrait

        ' PrintOptions to set Color
        If MyPrintJob.Printer.Color Then
            MyPrintJob.PrintOptions.Color = True
        End If

        ' Print the print job
        MyPrintJob.Print()
    End Sub
End Class
